plugins/services/src/js/containers/volume-detail/VolumeDetail.js

```diff
@@ -37,7 +37,7 @@ class VolumeDetail extends React.Component {
         <DetailViewHeader
           subTitle={this.renderSubHeader()}
           title={volume.getId()} />
-        <DescriptionList hash={detailsHash} />
+        <HashMapDisplay hash={detailsHash} />
```

Comment:
Same thing here: Instead of creating the configuration and then using the HashMapDisplay, could we just use ConfigurationMap-elements directly?